### PR TITLE
cmd/wit-bindgen-go: display module version and git revision, when available

### PR DESCRIPTION
**Pull request by ydnar**

### CODE REVIEW

File: cmd/wit-bindgen-go/main.go [hunk 1]
```diff
@@ -45,7 +51,7 @@ func main() {
 				Usage: "force loading WIT via wasm-tools",
 			},
 		},
-		Version: fmt.Sprintf("%v (%v)", version, revision),
+		Version: version,
```
Comment (Mossaka) [inline cmd/wit-bindgen-go/main.go:54]: I think it's always clearer to display the module version. Perhaps, we can remove git revision for tag version and display with untagged and dirty ones.

Comment (ydnar) [reply cmd/wit-bindgen-go/main.go:54]: Have you seen a case where both are present? (I haven’t, which is why I collapsed them)

Comment (Mossaka) [reply cmd/wit-bindgen-go/main.go:54]:
```bash
(base) ➜  ~ wasm-tools -V
wasm-tools 1.216.0 (28c8962b1 2024-08-22)
(base) ➜  ~ wit-bindgen -V                                  
wit-bindgen-cli 0.32.0 (dea39d07a 2024-09-10)
(base) ➜  ~ wasmtime -V                  
wasmtime-cli 24.0.0 (6fc3d274c 2024-08-20)
(base) ➜  ~ spin -V    
spin 2.4.2 (340378e 2024-04-03)
```

Comment (ydnar) [reply cmd/wit-bindgen-go/main.go:54]: Both are never present in this program. Either one or the other are present, including "devel".

Comment (Mossaka) [reply cmd/wit-bindgen-go/main.go:54]: Oh yeah I misunderstood your question. They were not present in this program. I am suggesting to add both version and revision like the examples that I gave.

Comment (ydnar) [reply cmd/wit-bindgen-go/main.go:54]: `debug.ReadBuildInfo` doesn’t have both version and revision.

Comment (ydnar) [reply cmd/wit-bindgen-go/main.go:54]:
```sh
❯ go install github.com/bytecodealliance/wasm-tools-go/cmd/wit-bindgen-go@main
go: downloading github.com/bytecodealliance/wasm-tools-go v0.2.2-0.20241001012006-bffa71f71f96

❯ wit-bindgen-go --version
wit-bindgen-go version v0.2.2-0.20241001012006-bffa71f71f96 ()
```

Comment (ydnar) [reply cmd/wit-bindgen-go/main.go]: Fixed!

**Pull request merged**